sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/TimeSeriesIdTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.TimeSeriesInsights.Tests
+{
+    public class TimeSeriesIdTests : E2eTestBase
+    {
+        private static readonly TimeSpan s_retryDelay = TimeSpan.FromSeconds(10);
+
+        // This is the GUID that TSI uses to represent the default type for a Time Series Instance.
+        // TODO: replace hardcoding the Type GUID when the Types resource has been implemented.
```

Comment:
would be good to add a todo task and add a number here to ensure we don't forget.

Reply:
Done. Added a task in the Types PBI

Reply:
If you are already making additional changes to this class, I would recommend adding a task number in comment as well next to TODO. That's what I did in previous PR.